### PR TITLE
Add Pull

### PR DESCRIPTION
**Pull request by wei**
As mentioned in probot/ideas#26

[Visit repo](https://github.com/wei/pull)

### CODE REVIEW

**Comment by wei** [comment]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;See no evil
I didn't break the build 🙈

**Comment by bkeepers** [comment]
> I didn't break the build 🙈

You did not, I did. 😭

File: _apps/pull.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Pull
+description: Keep your forks up-to-date.
+slug: pull
+# Include a few screenshots that show your app in action
```
Comment (hiimbex) [inline _apps/pull.md]: Could you delete this comment please?

File: _apps/pull.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: Pull
+description: Keep your forks up-to-date.
+slug: pull
```
Comment (hiimbex) [inline _apps/pull.md]: wow I can't believe no one else had this slug!

**Comment by hiimbex** [comment]
This time @JasonEtco broke the build :trollface:

**Pull request merged**